screens/SendSetupScreen.js

```diff
@@ -331,7 +346,99 @@ const SendSetupScreen = ({
     }
   };
 
+  const handleDeepLink = deepLinkParams => {
```

Comment:
prefer de-structuring in the function definition.  Might make some code like this a bit more verbose, but it makes the data flow much easier to follow, and keeps it consistent with the rest of the.  
Also, create a `DeepLinkParams` flow type declaration

ie
```javascript
handleDeepLink = ({address, tokenId, symbol}: DeepLinkParams) => {
...
```